docs/make.bat

```diff
@@ -0,0 +1,242 @@
+@ECHO OFF
```

Comment:
This file is new to me :) I guess it's meant to support Windows devs, but it looks kind of too hard to maintain, and the Windows devs also have options to use contentional Make :)